src/main/java/com/jcabi/github/RtCommitsComparison.java

```diff
@@ -94,4 +104,75 @@ public JsonObject json() throws IOException {
         return new RtJson(this.request).fetch();
     }
 
+    /**
+     * Iterator that yields FileChange objects converted
+     * from JSON objects in a JSON list.
+     */
+    @EqualsAndHashCode(of = { "iterator" })
+    @Loggable(Loggable.DEBUG)
+    @ToString
+    private static final class FileChangesIterator
+        implements Iterator<FileChange> {
+        /**
+         * Encapsulated iterator of file change JSON objects.
+         */
+        private final transient Iterator<JsonObject> iterator;
+
+        /**
+         * Ctor.
+         * @param iter Iterator of file change JSON objects
+         */
+        FileChangesIterator(
+            @NotNull(message = "files can't be NULL")
+            final Iterator<JsonObject> iter
+        ) {
+            this.iterator = iter;
+        }
+
+        @Override
+        public FileChange next() {
+            return new RtFileChange(this.iterator.next());
```

Comment:
FindBugs currently gives a false-positive warning here:
```
[WARNING] FindBugs: L B It: com.jcabi.github.RtCommitsComparison$FileChangesIterator.next() can't throw NoSuchElementException  At RtCommitsComparison.java:[line 134]
```
I don't know how to tell qulice to suppress this.